Create new issue template for new-fma.md (#43455)
fleetdm/fleet: .github/ISSUE_TEMPLATE/new-fma.md

> ## name: New FMA 

Reviewer: ```suggestion
## name: New Fleet-maintained app (FMA)
```

I think we want to be explicit and spell out the acronym in the title. This way, we help folks who are new to Fleet understand.

The title is significantly bigger than the description in the "New issues" list:

<img width="997" height="647" alt="Image" src="https://github.com/user-attachments/assets/fa616b51-6e92-4ce4-a874-a2f8cd45788a" />

> about: Request a new Fleet-maintained app for macOS or Windows

Reviewer: ```suggestion
about: Request a new FMA for macOS or Windows
```

> title: "New FMA: [APP NAME]"

Reviewer: ```suggestion
title: "New Fleet-maintained app: [APP NAME]"
```

labels: fma, :help-solutions-consulting
assignees: ''

## Description

### App details

- **Application name:** [e.g., Mozilla Firefox]
- **Application platform:** [macOS/Windows]

---

## Validation checklist

- [ ] `/outputs/<app-name>/<platform>.json` created
- [ ] `/outputs/apps.json` updated
- [ ] Manifest name matches osquery `app.name` (macOS) or `programs.name` (Windows)
- [ ] Manifest version scheme matches osquery `app.short_bundle_version` (macOS) or `programs.version` (Windows)
- [ ] Manifest `unique_identifier` matches osquery `app.bundle_identifier` (macOS only)

---

## QA checklist

- [ ] App adds successfully to team's library
- [ ] App installs successfully on host
- [ ] App opens successfully on host
- [ ] App uninstalls successfully on host

---

## Icon

- [ ] Icon added to Fleet
- [ ] Correct icon appears in the app catalog

---

## Additional Notes

> [Add any additional context, dependencies, or special instructions here.]

Reviewer: @allenhouchins do y'all need all this?

How can we make this as simple/fast as possible for customers/community?

I'm thinking all we need is the name of an app w/ a download URL? URL so we know exactly what app they're after.

Reviewer 2: All we should need to capture is the name of the app, the app's website or download URL so we are capturing the right one, and which platform they targeted. We may not need a template for this. We have nearly 300 FMAs now and its a finite list of what we can support.

Reviewer: Hey @GrayW we decided to update the description for the "Feature request" template instead: https://github.com/fleetdm/fleet/pull/43870

This way we avoid adding another issue template. We want to keep the number of templates down. The fewer we have, the easier it is to file an issue (choice overload)

Reviewer: FYI @ireedy
